app/password/reset/route.js

```diff
@@ -2,18 +2,26 @@ import Ember from 'ember';
 import DisallowAuthenticated from "../../mixins/routes/disallow-authenticated";
 
 export default Ember.Route.extend(DisallowAuthenticated, {
-  model: function(){
+  model() {
     return this.store.createRecord('password-reset-request');
   },
 
+  resetController(controller, isExiting) {
```

Comment:
Whoa- I used `willTransition`. This is better, I didn't realized it had been added.